StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -25,6 +25,10 @@ extension PaymentSheet {
                 return [.returnURL]
             case .dynamic("zip"):
                 return [.returnURL]
+            case .dynamic("grabpay"):
+                return [.returnURL]
+            case .dynamic("fpx"):
+                return [.returnURL]
```

Comment:
Should we put grabpay or fpx under 'dynamic' when we have bindings for them?  cc @wooj-stripe

Reply:
Oh good catch, I didn't realize this would work without it, but I confirmed they still do.